Tools/CISupport/ews-build/steps.py

```diff
@@ -6639,7 +6639,8 @@ def run(self, BufferLogObserverClass=logobserver.BufferLogObserver):
             rc = FAILURE
 
         if rc == FAILURE:
-            self.setProperty('comment_text', f"{self.summary}, blocking PR #{self.getProperty('github.number')}")
+            build_url = f'{self.master.config.buildbotURL}#/builders/{self.build._builderid}/builds/{self.build.number}'
+            self.setProperty('comment_text', f"{self.summary}, blocking PR #{self.getProperty('github.number')}. Details: {build_url}")
```

Comment:
Could we clean it up to be something like this instead of showing the full link?

> Details: [Build #1234](https://ews-build.webkit.org)

`url = '[Build #{}]({})'.format(self.getProperty("buildnumber", ""), build_url)`

Reply:
This is also closer to what `safe-merge-queue` does, and we should probably standardize the comments.

Reply:
Good idea. Done in updated PR.